source/frontend/src/features/leases/detail/LeasePages/payment/__snapshots__/PeriodPaymentsContainer.test.tsx.snap

```diff
@@ -1,274 +1,5 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`PaymentsView component > renders as expected 1`] = `
```

Comment:
I've compared with Dev and I see no actual difference in the rendering. I will check with Sue once this is merged to confirm that nothing else has changed